h/buildext.py

```diff
@@ -215,8 +235,12 @@ def build_chrome(args):
     # Render the sidebar html.
     if webassets_env.url.startswith('chrome-extension:'):
         build_extension_common(env, bundle_app=True)
+        request = env['request']
+        app_dict = {
```

Comment:
Not sure what `app_dict` means. `context` would be a more conventional name for a template context object :)